app/views/projects/automated_workflow_executions/_automated_workflow_execution.html.erb

```diff
@@ -5,6 +5,18 @@
   <td class="px-6 py-4 whitespace-nowrap">
     <%= automated_workflow_execution.id %>
   </td>
+  <td
+    class="
+      px-6
+      py-4
+      font-medium
+      text-gray-900
```

Comment:
**nitpick (blocking):** Shouldn't this be `text-slate-900`

Reply:
Updated in 20db5c7

Reply:
Updated in 20db5c7